### DISCUSSION TITLE
Bluetooth midi?

### DISCUSSION THREAD

**Discussion by Typro7** [onlaj/Piano-LED-Visualizer#261] [Q&A] [answered]
I was looking into getting a keyboard with midi Bluetooth capabilities, but not USB midi capabilities. Is there an easy way I could still connect to the Pi Zero W and still be able to use the keyboard as a midi on a laptop as well even though it doesn't have USB capabilities? I like the keyboard I'm looking at right now a lot, but I just don't know if it would work with this project.

**Comment by onlaj** [comment, marked as answer]
What keyboard is that? Maybe it has MIDI ports, so you can just buy a MIDI to USB interface.
If it only has bluetooth, you can connect it to your PC and then pass the midi signal to the RPi using RTP midi.
It should work, but I'm afraid that delay might be noticable.

**Reply by Typro7** [reply]
I don't believe the charging port is capable of midi output, so I don't know if that would work, but I found out that my really old but good keyboard has an old midi port so I can just get a 10 dollar adapter and use the old keyboard. I think I'll just stick with that one and everything will be simple and easy to follow. haha thanks for answering though!
